src/components/UI/Text/TimeInput.vue

```diff
@@ -66,6 +66,17 @@
       <p class="text-xs pl-2 place-self-center text-red-600">{{ invalidInputWarning }}</p>
 
     </div>
+
+    <div v-if="timeExceedsVideoDuration" class="flex flex-row pl-2">
```

Comment:
there should not be any connection between `video` and TimeInput right? It can be named: `maxValidTime`?